Makefile

```
@@ -167,6 +168,15 @@ unit-test: unit-tests
 int-tests: gotools fabric-ca-server fabric-ca-client
 	@scripts/run_integration_tests
 
+vendor: .FORCE
+	@rm go.mod
+	@touch go.mod
+	@mv vendor.sum go.sum
+	@go mod tidy -modfile vendor.mod
+	@go mod vendor  -modfile vendor.mod
+	@mv go.sum vendor.sum
+	@rm go.mod
+
```

I think we're getting to the point of diminishing returns but I don't think this is quite right.

The target is doing this:
```sh
$ make -n vendor
rm go.mod
touch go.mod
mv vendor.sum go.sum
go mod tidy -modfile vendor.mod
go mod vendor  -modfile vendor.mod
mv go.sum vendor.sum
rm go.mod
```

```suggestion
vendor: .FORCE
	@echo > go.mod
	@go mod tidy -modfile vendor.mod
	@go mod vendor  -modfile vendor.mod
	@rm go.mod

```

---

@sykesm thanks ... went with this